extensions/amp-sidebar/1.0/sidebar.jss.js

```diff
@@ -60,13 +63,19 @@ const defaultBackdropStyles = {
   backgroundColor: 'rgba(0, 0, 0, 0.5)',
 };
 
+const sidebarChildClass = {
+  height: '101vh !important',
+  width: '0 !important',
+};
+
```

Comment:
The backdrops child has a height that is larger than the backdrop to force the scroll behavior in the backdrop.